docs/install.md

```diff
@@ -121,18 +121,16 @@ For example, you can do the following to have Prettier run before each commit:
 
    ```bash
    npm install --save-dev husky lint-staged
-   npx husky install
-   npm pkg set scripts.prepare="husky install"
-   npx husky add .husky/pre-commit "npx lint-staged"
+   npx husky init
+   echo "npx lint-staged" > .husky/pre-commit
```

Comment:
I forgot this doesn't work on Windows...

Let's use 

```js
node --eval "fs.writeFileSync('.husky/pre-commit','npx lint-staged')"
```

See https://github.com/prettier/prettier/blob/c8ba8dbca18858a7962184bbb3898502b9ec7cfb/docs/install.md?plain=1#L31-L36

Reply:
Updated